terraform/README.md

```diff
@@ -0,0 +1,34 @@
+###**EFS**

```

Comment:
The documentation purpose is to instruct whoever will us your repository on how to use it.

This kind of documentation with links on how each resource work can be left to help people learn more about those resources, but it's not something critical to the repository documentation.

Reply:
Done